api/src/middleware/__tests__/competenciesRouter.ts

```diff
@@ -104,4 +109,80 @@ describe('competenciesRouter', () => {
         .expect(500, done);
     });
   });
+
+  describe('PUT /:id', () => {
+    it('should update a competency of user who authored it', done => {
+      const principalId = 2;
+      const competencyId = 3;
+      const label = 'label';
+      const description = 'description';
+      const competency = {
+        id: competencyId,
+      };
+      mockPrincipalId(principalId);
+      mockAuthorizeCompetencyUpdate.mockResolvedValue(true);
+      mockUpdateCompetency.mockResolvedValue();
+      appAgent
+        .put('/3')
+        .send({
+          label: label,
+          description: description,
+        })
+        .expect(200, itemEnvelope(competency), err => {
+          expect(mockAuthorizeCompetencyUpdate).toHaveBeenCalledWith(
+            principalId,
+            competencyId
+          );
+          expect(mockUpdateCompetency).toHaveBeenCalledWith(
+            competencyId,
+            label,
+            description
+          );
+          done(err);
+        });
+    });
+
+    it('should respond with a validation error if given id is less than 1', done => {
```

Comment:
The description says "validation error" but the code tests for `400`, a bad request error. Looks like the description of the test should be updated.

Reply:
Will update with `'should respond with a bad request error if the given id is less than 1'` message.